be/tx.c

```diff
@@ -740,6 +740,37 @@ M0_INTERNAL void m0_be_tx_gc_enable(struct m0_be_tx *tx,
 	tx->t_gc_param   = param;
 }
 
+M0_INTERNAL void m0_be_tx_lsn_set(struct m0_be_tx *tx,
+                                  m0_bindex_t      lsn,
+                                  m0_bindex_t      lsn_discarded)
+{
+	/*
+	 * This function is called from m0_be_tx_group_close().
+	 * When the BE tx group this transaction belongs to is being closed
+	 * the tx sm group is not locked - this is why BE_TX_LOCKED_AT_STATE
+	 * could not be used here.
+	 */
+	M0_PRE(m0_be_tx_state(tx) == M0_BTS_CLOSED);
+
+	tx->t_lsn = lsn;
+	tx->t_lsn_discarded = lsn_discarded;
+	M0_LEAVE("tx=%p t_lsn=%"PRIu64" t_lsn_discarded=%"PRIu64,
+	         tx, tx->t_lsn, tx->t_lsn_discarded);
```

Comment:
The compiler will optimize this maybe, but we can use: lsn, and lsn_discarded instead of tx->t_lsn, tx->t_lsn_discarded.

Reply:
The message tells t_lsn and t_lsn_discarded. If I put different variables it would be a lie.

Removed your change from `fdmi_int`.